test/method/testddec.py

```diff
@@ -164,3 +165,31 @@ def test_water_charges(self):
         )
         # Check assigned charges
         assert_allclose(analysis.fragcharges, [-0.757097, 0.378410, 0.378687], atol=0.2)
+
+    def test_chgsum_h2(self):
+        """ Are DDEC6 charges for hydrogen atoms in nonpolar H2 small as expected?
+        """
+
+        h2path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "h2.out")
+        data = ccread(h2path)
+        vol = volume.Volume((-2, -2, -2), (2, 2, 2), (0.2, 0.2, 0.2))
+        analysis = DDEC6(data, vol, os.path.dirname(os.path.realpath(__file__)))
+        analysis.calculate()
+
+        self.assertAlmostEqual(numpy.sum(analysis.fragcharges), 0, delta=1e-2)
+        self.assertAlmostEqual(analysis.fragcharges[0], analysis.fragcharges[1], delta=1e-6)
```

Comment:
Come to think of it, is it expected that these charges are both
1. non-negligible and
2. the same sign?
It should actually be the `sum` test that has a very tight threshold, and the opposite test would be comparing the absolute value of the difference to zero.

Reply:
The charges for H2 should just be zero. Using much denser grid (spacing of 0.1 rather than 0.2 which is the cube file included in the test) gives [0.00046066, 0.00046066].

I've modified this test so that the delta is smaller, and left a note why the values are not exactly zero in the test.